### PR TITLE
Improve tiny mce detection

### PR DESCRIPTION
**Pull request by kkadosh**
for example we can find : `<script src="/Scripts/tiny_mce/tiny_mce.js"></script>`

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -9880,7 +9880,7 @@
       "js": {
         "tinyMCE.majorVersion": "([\\d.]+)\\;version:\\1"
       },
-      "script": "/tinymce(?:\\.min)?\\.js",
+      "script": "/tiny(?:_)?mce(?:\\.min)?\\.js",
```
Comment (jvoisin) [inline src/apps.json]: What about `"script": "/tiny_?mce(?:\\.min)?\\.js",` instead?

**Comment by kkadosh** [comment]
Is it possible to restart the CI ?

**Comment by jvoisin** [comment]
Just do a dummy commit :)

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**